### PR TITLE
Tweak documentation for adding a parser

### PR DESCRIPTION
**Pull request by grunweg**
Thanks a lot for the extensive documentation, this made it pretty easy to add two new parsers for languages I care about.

I noticed two small things in the documentation; updating it to reflect this. See commit messages for details.

### CODE REVIEW

File: manual/src/adding_a_parser.md [hunk 1]
```diff
@@ -93,10 +93,13 @@ language.
 "json" => Some(Json),
 ```
 
-There may also file names or shebangs associated with your
-language. [GitHub's linguist
+There may also file names or shebangs associated with your language. Add those by changing the `LANG_EXTENSIONS` constant further below in that file. GitHub's linguist definitions is a useful source of common file extensions.
+
+
+There may also file names or shebangs associated with your language.
```
Comment (Wilfred) [inline manual/src/adding_a_parser.md:99]: Looks like you've duplicated a sentence here?

Comment (grunweg) [reply manual/src/adding_a_parser.md:96]: Whoops, indeed. Fixed.

**Comment by grunweg** [comment]
Found two more places where the docs were out of date; updated accordingly. (And fixed up my second edit, as I noticed the grammar was still a bit off.)

**Comment by Wilfred** [comment]
Thanks.

**Pull request merged**